apps/src/studio/blocks.js

```diff
@@ -3717,7 +3492,31 @@ exports.install = function (blockly, blockInstallOptions) {
     removeVar: Blockly.Blocks.variables_get.removeVar,
   };
 
-  generator.studio_ask = function () {
+  // Overrides the standard generator from Core Blockly.
+  // Variable labels in Playlab include the Globals namespace.
+  blockGeneratorFunctionDictionary.variables_get = function () {
```

Comment:
These new generator functions come CDO Blockly: https://github.com/code-dot-org/blockly/blob/v4.1.0/generators/javascript/variables.js#L30-L46

Google Blockly's are here: https://github.com/google/blockly/blob/505492526c15817a4ecc09d6f36820fff3be5136/generators/javascript/variables.ts

We only need to main this set for Play Lab, which is the only lab that relies upon `translateVarName` (to add the `Globals` namespace).